Tests/MessagingPush/IntegrationTest.swift

```diff
@@ -26,4 +27,18 @@ class IntegrationTest: SharedTests.IntegrationTest {
             MessagingPush.initialize()
         }
     }
+
+    func getPush(content: [AnyHashable: Any], deliveryId: String = .random, deviceToken: String = .random) -> CustomerIOParsedPushPayload {
```

Comment:
This is not a new function. It got moved into this file so that any integration test within MessagingPush module can use it.